src/PanelTraits/Update.php

```diff
@@ -73,4 +75,31 @@ public function getUpdateFields($id)
 
         return $fields;
     }
+
+    /**
+     * Get the value of the 'name' attribute from the declared relation model in the given field.
+     *
```

Comment:
Ok, for the life of me I can't picture what this does.

Can we have an example in the doc block?